README.md

```diff
@@ -1,2 +1,31 @@
-# HelloMicroverse
-First repository at Microverse
+# Hello-Microverse
+
+- First repository setup at Microverse. Added the html/css linter to the repository.
+
+## Built with
+
+- HTML
+- CSS
+
+## Setup for Basic HTML/CSS Project
+
+- Linter for HTML/CSS installed
+
+## Getting Started
```

Comment:
Hello @itsmraga-hub. You are really doing a great job.✔️ 

- [ ] However, you may try to customize your readme file to look similar to the version [Readme File](https://github.com/microverseinc/readme-template/blob/master/README.md). This will make your project look more professional.